test/deploy-cluster.sh

```diff
@@ -92,7 +92,9 @@ else
   # reference: https://github.com/kubeflow/pipelines/issues/6696
   # Hard-coded GKE to 1.25.10-gke.1200 (the latest 1.25 in STABLE channel). Reference: 
   # https://github.com/kubeflow/pipelines/issues/9704#issuecomment-1622310358
-  gcloud container clusters create ${TEST_CLUSTER} --image-type cos_containerd --release-channel stable --cluster-version 1.25.10-gke.1200 ${SCOPE_ARG} ${NODE_POOL_CONFIG_ARG} ${WI_ARG}
+  # 08/09/2023 update: 1.25.10-gke.1200 no longer supported, use 1.25.10-gke.2100 instead. Reference:
```

Comment:
Can we still specify 1.25 explicitly? In that way, when the default changed to 1.26+, the error message would be more clear and strait-forward instead of being "daemon connection failed".

https://cloud.google.com/kubernetes-engine/versioning#specifying_cluster_version

Reply:
Thank you for providing the info on ways we can specify cluster version, it's really helpful! Using 1.25 would make the test cluster much more reliable. I will change the version and rerun the test.